src/public/javascripts/resperson_utils.js

```diff
@@ -0,0 +1,66 @@
+$(document).ready(function () {
+    $('th').each(function (col) {
+        $(this).hover(function () {
+            $(this).addClass('focus');
+        }, function () {
+            $(this).removeClass('focus');
+        });
+        $(this).click(function () {
```

Comment:
`Question`: Where is the `asc`/`desc` code used?
* I can't find it.

Reply:
This code is used to sort when one click on the table header, this is used though out ex. for people, crafts, org users etc.

Reply:
This makes sense, I did not read the code properly, thanks.